docs/release/trg-0/trg-1-09.md

```diff
@@ -0,0 +1,42 @@
+---
+title: TRG 1.09 - UX consistency / Styleguide for UI
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Draft  | 10-Jan-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+Ensuring UX consistency through a style guideline is essential for a unified and user-friendly experience across front end modules.
+
+## Description
+
+- User Interfaces **should** align with the Tractus-X Style Guidelines
+
+> User interface style review has been executed.
+> Review feedback (if existing) got incorporated.
+> All findings are assessed.
+> All findings (high/very high) are fixed or cleaned up (evidence by re-review).
+> Approval of the application CX Style conformity is available (given by the review owner).
```

Comment:
Avoid abbriviations, use full name instead if needed for Catena-X.

Reply:
The > sections are 'reminder textes' which need to be rewriten by Julia